test/e2e/windows/density.go

```diff
@@ -30,21 +30,14 @@ import (
 	"k8s.io/apimachinery/pkg/watch"
 	"k8s.io/client-go/tools/cache"
 	"k8s.io/kubernetes/test/e2e/framework"
-	imageutils "k8s.io/kubernetes/test/utils/image"
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 )
 
 var _ = SIGDescribe("[Feature:Windows] Density [Serial] [Slow]", func() {
-
 	f := framework.NewDefaultFramework("density-test-windows")
 
-	BeforeEach(func() {
-		// NOTE(vyta): these tests are Windows specific
-		framework.SkipUnlessNodeOSDistroIs("windows")
-	})
```

Comment:
Note to reviewers: this has been moved to https://github.com/kubernetes/kubernetes/pull/74738/files#diff-b0ddb1c276f901c22122f6283cc7348c